src/main/java/com/algorand/algosdk/templates/ContractTemplate.java

```diff
@@ -14,44 +14,66 @@
 
 public class ContractTemplate {
 
-	public enum DataType {
-		BASE64, ADDRESS, INT;
+	/**
+	 * Values are appended to a program at specific offsets. Depending on the type the source template has placeholders
+	 * of differing sizes, this offset is returned by the "updateProgram" method and is used to traverse the template.
+     *
+	 * Offsets are the following:
```

Comment:
I do not think the use of the term "Offsets" here is correct. The following of the placeholder sizes, and not offsets.

Reply:
Yes, you're right. I've tried to clarify this.